package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webmaker.org",
-  "version": "1.1.150",
+  "version": "1.1.151",
```

Comment:
[the latest tag](https://github.com/mozilla/webmaker.org/tags) is v1.1.152 I wonder if `version` should be set to 1.1.152 here?